typhos/positioner.py

```diff
@@ -298,9 +377,31 @@ def add_device(self, device):
         self._link_low_limit_switch()
         self._link_high_limit_switch()
 
+        try:
+            if device.stop is None:
```

Comment:
`ophyd.Device` is guaranteed to have `stop` as an attribute. The guard below against `AttributeError` feels appropriate to me regardless.

Reply:
Paranoia as a policy of course